input-stepper/doc/index.md

```diff
@@ -17,16 +17,20 @@ Click the flip locale button to see localized default label in action as well as
 
 ```js preview-story
 export const input = () => html`
-  <button @click=${() => {
-    if (localize.locale === 'en-GB') {
-      localize.locale = 'nl-NL';
-    } else {
-      localize.locale = 'en-GB';
-    }
-  }}>flip locale</button>
-  <simba-input-stepper 
+  <button
+    @click=${() => {
+      if (localize.locale === 'en-GB') {
+        localize.locale = 'nl-NL';
```

Comment:
NL 🤣. More seriously looks like a toggle to me, doesn't it ?

Reply:
Yeahhh it's a very basic not so concise toggler to go between nl-NL and en-GB :P